lib/matplotlib/axes/_axes.py

```diff
@@ -2259,7 +2259,7 @@ def barh(self, bottom, width, height=0.8, left=None, **kwargs):
             dictionary of kwargs to be passed to errorbar method. `ecolor` and
             `capsize` may be specified here rather than as independent kwargs.
 
-        align : ['edge' | 'center'], optional, default: 'edge'
+        align : ['edge' | 'center'], optional
```

Comment:
Can you change this to use the format of `bar` with braces? The default is the first item, so change the order.